notebooks/idr0002_zarr.ipynb

```diff
@@ -25,7 +37,17 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"JAVA_HOME\"]=\"/Users/jmarie/opt/anaconda3/envs/cp_test\""
```

Comment:
Yes, I have the latest version, and this line persists

Reply:
found it. sorted now